src/System.Private.CoreLib/shared/System/TimeZoneInfo.cs

```diff
@@ -51,7 +51,7 @@ private enum TimeZoneInfoResult
         private const string UtcId = "UTC";
         private const string LocalId = "Local";
 
-        private static readonly TimeZoneInfo s_utcTimeZone = CreateCustomTimeZone(UtcId, TimeSpan.Zero, UtcId, UtcId);
+        private static readonly TimeZoneInfo s_utcTimeZone = CreateCustomTimeZone(UtcId, TimeSpan.Zero, "(UTC) Coordinated Universal Time", "Coordinated Universal Time");
```

Comment:
Is it correct that these are not localized?

Reply:
We didn't localize the display name for the UTC before as you see the old was doing the same. We can try to fix that in other change.